src/status_im/ui/screens/desktop/main/chat/views.cljs

```diff
@@ -56,9 +58,12 @@
        [react/text {:style styles/author} name]])))
 
 (views/defview member-photo [from]
-  [react/view
-   [react/image {:source {:uri (identicon/identicon from)}
-                 :style  styles/photo-style}]])
+  [react/touchable-highlight {:on-press #(do
+                                           (re-frame/dispatch [:set-public-chat-whisper-identity from])
```

Comment:
we're trying to do not dispatch more than one event, so better to create a new event and change app-db directly

Reply:
Awesome, thanks for this feedback @flexsurfer. I've updated to act on `app-db` directly.